config/locales/en.yml

```diff
@@ -92,6 +92,8 @@ en:
     1400:
       103: 'Channel Islands duty'
       105: 'Channel Islands duty'
```

Comment:
I am wondering if we should also update these to "UK-CD Customs Union"

Reply:
Okay, ignore this - we're just wayyyy ahead of CDS people and these measures are coming soon